ngshare/ngshare.py

```diff
@@ -291,11 +318,23 @@ def post(self, course_id, instructor_id):
         course = self.find_course(course_id)
         self.check_course_instructor(course)
         instructor = self.find_user(instructor_id)
+        first_name = self.get_argument('first_name', None)
+        if not first_name:
+            self.json_error('Please supply first name')
+        last_name = self.get_argument('last_name', None)
+        if not last_name:
+            self.json_error('Please supply last name')
+        email = self.get_argument('email', None)
+        if not email:
+            self.json_error('Please supply email')
```

Comment:
@aalmanza1998  Should empty strings be allowed?

Reply:
Your choice. FYI currently this does not allow empty string, but allowing it is a trivial fix.

Reply:
Right now first name, last name, and email are optional to add a student on nbgrader. Only the student ID is required. I'm not sure if you want to keep this functionality. I don't know why we would want those to be optional? if you have any suggestions as to when first name, last name and email should be optional, then empty strings should be allowed. Otherwise, they should not.

Reply:
Conclusion: allow empty strings